vlasovsolver/cpu_1d_pqm.hpp

```diff
@@ -99,7 +99,11 @@ inline void filter_pqm_monotonicity(Vec *values, uint k, Vec &fv_l, Vec &fv_r, V
       //todo store and then load data to avoid inserts (is it beneficial...?)
       
 //serialized the handling of inflexion points, these do not happen for smooth regions
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunknown-pragmas"
 #pragma ivdep
+#pragma GCC diagnostic pop
+#pragma GCC ivdep                     
```

Comment:
That's a lot of work for a little fix! :D

Reply:
I thought the same. Do we still target Intel compilers? If so, we could define a macro to use the correct `ivdep` `#pragma`.